src/main/java/com/dnsimple/Dnsimple.java

```diff
@@ -1,7 +1,7 @@
 package com.dnsimple;
 
 public abstract class Dnsimple {
-  public static final String VERSION = "0.4.1";
+  public static final String VERSION = "0.6.1";
```

Comment:
Please don't change the version, it is part of the release process.
https://github.com/dnsimple/dnsimple-java/blob/master/CONTRIBUTING.md#releasing

We can try a release immediately after so that we check also the PGP dependency.